vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -0,0 +1,193 @@
+- name: ensure web and proxy ports
+  become: yes
+  firewalld:
+    immediate: yes
+    permanent: yes
+    port: "{{ item }}"
+    state: enabled
+  loop:
+    - "80/tcp"
+    - "443/tcp"
+    - "488/tcp"
```

Comment:
Do we really need to open port 488 itself?  I think the FOIA proxy config should be enough, so 488 doesn't have to be open in the firewall?

Reply:
Thank you @david-oc-miller , you are right. Actually we don't need to open port 488